packages/catalog-client/src/types.ts

```diff
@@ -53,3 +53,17 @@ export type AddLocationResponse = {
 export type DiscoveryApi = {
   getBaseUrl(pluginId: string): Promise<string>;
 };
+/**
+ * This is a copy of the core IdentityApi, to avoid importing core.
+ */
+export type ProfileInfo = {
+  email?: string;
+  displayName?: string;
+  picture?: string;
+};
+export type IdentityApi = {
+  getUserId(): string;
+  getProfile(): ProfileInfo;
+  getIdToken(): Promise<string | undefined>;
```

Comment:
Can we rename this to get `authorizationToken` instead ? It could be an accessToken or an IdToken, both of which can be used for authorization.

Reply:
I don't mind particularly, I will adhere to whatever the core team suggests.